libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -159,9 +161,23 @@ const AP_Param::GroupInfo AP_Airspeed::var_info[] = {
     // @Param: _OPTIONS
     // @DisplayName: Airspeed options bitmask
     // @Description: Bitmask of options to use with airspeed.
-    // @Bitmask: 0:Disable on sensor failure,1:Re-enable on sensor recovery
+    // @Bitmask: 0:Disable airspeed sensor if difference between airspeed and ground speed is larger than ARSPD_WIND_MAX, if set 1:Re-enable on sensor if difference between airspeed and ground speed falls below ARSPD_WIND_MAX
```

Comment:
Can't have a comma in here:
```suggestion
    // @Bitmask: 0:Disable airspeed sensor if difference between airspeed and ground speed is larger than ARSPD_WIND_MAX; if set 1:Re-enable on sensor if difference between airspeed and ground speed falls below ARSPD_WIND_MAX
```

Reply:
longer description would be good, and shorter Bitmask descriptions

Reply:
fixed